src/apps/js_generic/js_generic.cpp

```diff
@@ -290,8 +290,14 @@ namespace ccfapp
 
       auto scripts = tx.get_view(this->network.app_scripts);
       scripts->foreach([&out](const auto& key, const auto&) {
-        out.methods.push_back(key);
-        return true;
+        size_t s = key.find(' ');
+        if (s != std::string::npos)
+        {
+          out.endpoints.push_back(
+            {key.substr(0, s), key.substr(s + 1, key.size() - (s + 1))});
+          return true;
+        }
+        return false;
```

Comment:
I was doing that while testing, but it didn't catch any, so yes, defaulting to POST is ok. It's fairly temporary anyway, I think I want the keys to be structured rather than space separated, and we can do that when we make the handlers independent.